zoo_bot/SensorData[] sensorData
zoo_bot/GeoPosition robotPos
float32 velocity
float32 batteryLevel